resources/views/home.blade.php

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" href="{{ URL::asset('favicon.png') }}" type="image/x-icon" />
+  <title>PHPNews | Home</title>
+  <link href="{{ asset('css/app.css') }}" rel="stylesheet">
```

Comment:
`URL::asset` giống với `asset` nhỉ? Thống nhất dùng 1 kiểu thôi nhé

Với những file nào được process bằng `mix` thì e nên dùng helper `mix` thay cho `asset`, dùng mix nó có thể generate ra url với version của file
https://laravel.com/docs/8.x/mix#versioning-and-cache-busting

Reply:
@tuanpt-0634 

- Ignore public/js, public/css
- Dùng `mix()` thay `assets()`;  `assets()` thay `URL::asset()`
<br/>
Em update rồi ạ (bow)